mars/services/task/supervisor/processor.py

```diff
@@ -463,6 +493,12 @@ def get_result_tileables(self):
             result.append(build_fetch(tiled))
         return result
 
+    def get_result_tileable_graph(self):
```

Comment:
Rename as `get_tileable_graph_as_dict` as the original name may imply that the return value is a `TileableGraph`.